README.md

```diff
@@ -19,7 +19,7 @@
 # Overview
 
 This project aims to be a full port of the original C# [Neo project](https://github.com/neo-project).
-A complete toolkit for the NEO blockchain, including:
+A complete toolkit for the NEO blockchain includes the following:
```

Comment:
I'd leave it as is, semantically that's what we want to say --- we've got a complete thing, including the following ingredients.